client/web/src/pages/Conversation.jsx

```diff
@@ -44,10 +45,20 @@ const Conversation = ({
   selectedCharacter,
   messageId,
   token,
+  isTextStreaming,
   sessionId,
 }) => {
   const navigate = useNavigate();
 
+  const message = isTextStreaming ? '' : textAreaValue;
```

Comment:
this would use all text values so far in the conversation. Would messageInput be more relevant for emotion extraction?

Reply:
we have added a new prompt for Steve Jobs making the LLM returns emotion inside the bracket, then the emotion can be used to control the expression of the avatar.
But 'messageInput' only contains words from user,
so we need to extract the emotion it returns which can get from 'textAreaValue'.

Also the 'extractEmotionFromPrompt' function will automatically find the last emotion inside bracket.